lib/__tests__/config.test.ts

```diff
@@ -88,11 +88,58 @@ describe('Config object', () => {
 
   it("ignores an empty 'scopes' when isCustomStoreApp is true", () => {
     validParams.isCustomStoreApp = true;
+    validParams.adminApiAccessToken = 'token';
     delete (validParams as any).scopes;
 
     expect(() => validateConfig(validParams)).not.toThrow(
       ShopifyErrors.ShopifyError,
     );
+    validParams.isCustomStoreApp = false;
+  });
+
+  // ENABLE THIS TEST PRIOR TO NEXT MAJOR RELEASE
```

Comment:
This might be a neat opportunity to add a test helper that's version-aware! So it'll skip this test prior to v8 but run it afterwards.